CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Change Log
 
+## 2.9.0
```

Comment:
Will we be adding a date once the release date is known?

Reply:
Yes.